packages/components/src/molecules/SearchProducts/SearchProductItemContent.tsx

```diff
@@ -12,23 +13,63 @@ export interface SearchProductItemContentProps {
    * Specifies product's prices.
    */
   price: PriceDefinition
+  /**
+   * Quick order settings.
+   */
+  quickOrder?: {
+    enabled: boolean
+    availability: boolean
+    hasVariants: boolean
+    skuMatrixControl: React.ReactNode
+    quantity: number
+    onChangeQuantity(value: number): void
+    buyProps?: {
+      onClick: (e: React.MouseEvent<HTMLButtonElement>) => void
+      'data-testid': string
+      'data-sku': string
+      'data-seller': string
+    }
+  }
 }
 
 const SearchProductItemContent = forwardRef<
   HTMLElement,
   SearchProductItemContentProps
->(function SearchProductItemContent({ price, title, ...otherProps }, ref) {
+>(function SearchProductItemContent(
+  { price, title, quickOrder, ...otherProps },
+  ref
+) {
+  const renderProductItemContent = useCallback(() => {
+    return (
+      <>
+        <p data-fs-search-product-item-title>{title}</p>
+        {price.value !== 0 && (
+          <ProductPrice
+            data-fs-search-product-item-prices
+            listPrice={price.listPrice}
+            value={price.value}
+            formatter={price.formatter}
+          />
+        )}
+      </>
+    )
+  }, [quickOrder?.enabled])
```

Comment:
Typescript will complain about the `quickOrder` variable not being used inside the `useCallback` scope. Maybe by creating a function outside of the `SearchProductItemContent` scope can solve this, because the conditions at lines 60 and 62 will already prevent the unexpected re-renderings.